mimesis/builtins/pl.py

```diff
@@ -0,0 +1,100 @@
+"""Specific data provider for Poland (pl)."""
+
+from calendar import isleap
+from typing import Optional
+
+from mimesis.builtins.base import BaseSpecProvider
+from mimesis.enums import Gender
+from mimesis.typing import DateTime
+
+__all__ = ['PolandSpecProvider']
+
+
+class PolandSpecProvider(BaseSpecProvider):
+    """Class that provides special data for Poland (pl)."""
+
+    class Meta:
+        """The name of the provider."""
+
+        name = 'poland_provider'
+
+    def nip(self) -> str:
+        """Generate random valid 10-digit NIP.
+
+        :return: valid 10-digit NIP
+        """
+        nip_digits = list(map(int, str(self.random.randint(101, 998))))
+        for _ in range(6):
+            nip_digits.append(self.random.randint(0, 9))
+        args = (6, 5, 7, 2, 3, 4, 5, 6, 7)
```

Comment:
Bad name for the variable. What is actually this?

Reply:
`args` are special coefficients which are multiplied with corresponding digits in `nip_digits` in order to generate checksum digit for NIP.

Reply:
Can we use more informative name?

Reply:
Is `nip_coeffecients` ok?